sleap_nn/config/data_config.py

```
@@ -5,7 +5,7 @@
 """
 
 from attrs import define, field, validators
-from omegaconf import MISSING
+from omegaconf import MISSING, MissingMandatoryValue
```

Ruff (0.8.2)

`omegaconf.MissingMandatoryValue` imported but unused

Remove unused import: `omegaconf.MissingMandatoryValue`

(F401)